src/payment/cart/OrderSummary.jsx

```diff
@@ -12,6 +13,12 @@ class OrderSummary extends React.Component {
       'edx.bi.ecommerce.payment_mfe.order_summary_rendered',
       getPerformanceProperties(),
     );
+    let eventData = {};
+    let renderTime = window.performance.getEntriesByName("Order Summary component rendered");
+    if (renderTime) {
+      eventData["summaryRenderTiming"] = renderTime[renderTime.length - 1].toJSON();
```

Comment:
what is this?

Reply:
That gets the timing mark for when the Order Summary component finishes rendering (if available; taking the last one if multiple, which there shouldn't be, but `getEntriesByName` returns an array)

Reply:
so what happens if renderTime is not defined? do we not have a time? Should the sendEvent be inside this if block then?

Reply:
If we don't have a `renderTime`, oh well, but we should still send the event anyway so we can compare with the regular track event.